File: Back End/Demo.Entities/Pregunta.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Demo.Entities
{
    public class Pregunta:EntityBase
    {
        [Required]
        [Column(TypeName = "varchar(150)")]
        public string Descripcion { get; set; }
        public int CuestionarioId { get; set; }
        public Cuestionario? Cuestionario { get; set; }
        //cuidado debe ser exactamente igual aunque comienze con minuscula
        public List<Respuesta> listRespuestas { get; set; }
    }
}

File: FrontEnd/src/app/models/pregunta.ts
import { Respuesta } from "./respuesta";

export class Pregunta{
    descripcion?:string;
    listRespuestas?:Respuesta[];
    hide?:boolean;
    constructor(descripcion:string,listrespuestas:Respuesta[]){
        this.descripcion=descripcion;
        this.listRespuestas=listrespuestas;
        this.hide=true;
    }
}